src/redprl/operator.sml

```diff
@@ -482,8 +487,11 @@ struct
     fun spanToString f (r, r') =
       P.toString f r ^ " ~> " ^ P.toString f r'
 
-    fun extentsToString f =
-      ListSpine.pretty (P.toString f) ","
+    fun equationToString f (r, r') =
+      P.toString f r ^ "=" ^ P.toString f r'
```

Comment:
@jonsterling [extremely minor isssue] Do you want spaces before and after the equal sign?